apis/python/src/tiledbsc/assay_matrix.py

```diff
@@ -108,6 +108,46 @@ def df(self, obs_ids=None, var_ids=None) -> pd.DataFrame:
         """
         return self.dim_select(obs_ids, var_ids)
 
+    # ----------------------------------------------------------------
```

Comment:
I don't know which code style we're adhering to, but these two methods could be deduplicated by using a private `_to_sparse` method that accepts a matrix type argument. If we prefer to be more explicit though, this is totally fine.